src/src/algo/hillel/AlgoInsert.java

```diff
@@ -0,0 +1,34 @@
+package algo.hillel;
+
+import java.util.Arrays;
+
+public class AlgoInsert {
+    public static void main(String[] args) {
+        var ints =doArray(100);
+        System.out.println(Arrays.toString(ints));
+        var sort = insertSort(ints);
+        System.out.println(Arrays.toString(ints));
+    }
+    private  static  int[] insertSort(int[] ints){
+        for(int i=1;i<ints.length;i++){
+            int current = ints[i];
+            int j =i;
```

Comment:
`j` is supposed to be = `i - 1`
Here you're comparing the same element, because `i = 1` and `j = i`